lib/instrument-router.js

```diff
@@ -0,0 +1,47 @@
+const { Router } = require('express');
```

Comment:
this whole file should live in a folder  where all the route files will live
something like 

lib/routes/instrument-router.js